test/integration/smoke/test_vm_life_cycle.py

```diff
@@ -918,7 +918,7 @@ def secure_all_hosts(self):
             cmd = provisionCertificate.provisionCertificateCmd()
             cmd.hostid = host.id
             cmd.reconnect = True
-            self.apiclient.updateConfiguration(cmd)
+            self.apiclient.provisionCertificate(cmd)
```

Comment:
this one is already addressed in #2754 @rhtyd

Reply:
okay